src/bin/gpcheckhdfs/gpcheckhdfs.c

```diff
@@ -271,6 +273,21 @@ int testHdfsConnect(hdfsFS * fsptr, const char * host, int iPort,
     return 0;
 }
 
+/*
+ * check path is a trash directory
+ * path, e.g: /hawq_default/.Trash
+ */
+static int is_trash_directory(const char *path) {
```

Comment:
I suggest the name of this function uses "isTrashDirectory" instead of current one, since the other functions name follow this style: testFooBar, testXxxYyy, etc.

Reply:
thanks, I will fix it later.